examples/asa-vault/redemptions.js

```diff
@@ -0,0 +1,45 @@
+const {VaultClient, Vault, VAULT_IDS} = require('../..');
+const {ask} = require('@reach-sh/stdlib');
+const dotenv = require('dotenv');
+dotenv.config();
+
+(async () => {
+  const mnemonic = process.env.MNEMONIC;
+  const VAULT_ID = process.env.VAULT_ID ? process.env.VAULT_ID :
+  VAULT_IDS.TestNet.algo; VAULT_IDS.TestNet.algo;
+  const proposedAddress = process.env.ADDRESS_FOR_REDEMPTION;
+  const account = new VaultClient({
+    mnemonic,
+    network: 'TestNet',
+    asaVault: {decimals: 8},
+  });
```

Comment:
```suggestion
  const account = new VaultClient({
    mnemonic,
    network: 'TestNet',
    asaVault: {decimals: 8},
  }, true);
```
I think this needs to include the second parameter to define the `asa` vault? Alternatively, we could check the existence of  `params.asaVault` in the constructor instead.

Reply:
Looks like `VaultClient` uses this pattern but `Vault` does not - is there a reason for this?

Reply:
Thanks @AwolDes 
I just thought it was unnecessary since we did not need the decimals there but have made it consistent now